manager/src/main/java/com/redhat/service/bridge/manager/exceptions/AlreadyExistingItemException.java

```diff
@@ -4,6 +4,8 @@
 
 public class AlreadyExistingItemException extends EventBridgeManagerException {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
Do we need `serialVersionUID` in the exception classes?

Reply:
I did that to avoid an annoying warning that Eclipse displays because since Exception implements Serializable, theoretially it is a good practise to put a defaul serial in all the children. In practise, you are never going to serialize an Exception once RMI has dead for good, but nevertheless I prefer to put the line and avoid the yellow color in my screen ;)